shared/actions/kbfs/index.platform.desktop.js

```diff
@@ -42,22 +76,23 @@ function * openInWindows (openPath: string): SagaGenerator<any, any> {
     kbfsPath = yield call(kbfsMountGetCurrentMountDirRpcPromise)
 
     if (!kbfsPath) {
-      console.warn('Error in finding kbfspath')
-      return
+      throw new Error('No kbfsPath')
```

Comment:
:sheep: This check is kinda confusing to me. If `kbfsPath === Constants.defaultKBFSPath` how could `!kbfsPath` be true? Alternatively could move up the `if (!kbfsPath) ...` check below to above `kbfsPath === Constants.defaultKBFSPath`.